Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +114,27 @@ type State = {
 };
 type Event = Object;
 
+if (Platform.OS === 'android') {
+  var AndroidTextInput = requireNativeComponent('AndroidTextInput', null);
```

Comment:
What do you think about using `instance.viewConfig.uiViewClassName`? If that seems safe it wouldn't require any additional changes.